README.md

```diff
@@ -95,6 +95,12 @@ def row_by_value(df, col, value):
     """Slice out row from DataFrame by a value."""
     return df[df[col] == value].squeeze()
 
+# register_series_or_dataframe_method can optionally include arguments to pass to pd.DataFrame.apply()
+@pf.register_series_or_dataframe_method(axis=1)
+def total_diff(col, start_idx=0, end_idx=-1):
+  """ Return difference between the last and the first values per row/column"""
+  return col.iloc[end_idx] - col.iloc[start_idx]
```

Comment:
I know this is really not a great example of the need for this, as `total_diff` could easily be vectorized. I do believe this function is beneficial in situations where the operation cannot be vectorized and must be completed column by column. 

I am open to suggestions for a better example and will update the README once a better example is found.